### ISSUE TITLE
Try to train by COCO dataset

### ISSUE THREAD

**Issue by ghost** [xuannianz/keras-CenterNet#45]
I tried

`python train.py --freeze-backbone --gpu 0 --random-transform --compute-val-loss --batch-size 32 --steps 1000 coco datasets/coco`

but got AssertError
```
assert annotations['bboxes'].shape[0] != 0
AssertionError
```

Looks like not found some annotation

**Comment by ghost** [comment]
find one error in code
common.py 416
add 
 cls_id = int(cls_id)

**Comment by Salary-only-17k** [comment]
i have the same issue, how to do it? @vladimircape

**Comment by chenglong0317** [comment]
他的代码有问题或者说 没有制作成代码需要的数据集合。from pycocotools.coco import COCO用这个加载就好吧，把他里面的加载数据替换掉
